src/translations/translations.fr.js

```diff
@@ -483,6 +483,69 @@ export const translations = {
   //#endregion
+  //#region cc-orga-member-card
+  'cc-orga-member-card.btn.cancel.accessible-name': ({ memberIdentity }) => `Annuler modification du membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.cancel.visible-text': `Annuler`,
+  'cc-orga-member-card.btn.delete.accessible-name': ({ memberIdentity }) => `Supprimer le membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.delete.visible-text': `Supprimer`,
+  'cc-orga-member-card.btn.edit.accessible-name': ({ memberIdentity }) => `Modifier le membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.edit.visible-text': `Modifier`,
+  'cc-orga-member-card.btn.leave.accessible-name': `Quitter l'organisation`,
+  'cc-orga-member-card.btn.leave.visible-text': `Quitter`,
+  'cc-orga-member-card.btn.validate.accessible-name': ({ memberIdentity }) => `Valider la modification du membre - ${memberIdentity}`,
+  'cc-orga-member-card.btn.validate.visible-text': `Valider`,
+  'cc-orga-member-card.current-user': `Votre compte`,
+  'cc-orga-member-card.error.last-admin': () => sanitize`<p><strong>Il s'agit du dernier admin de l'organisation</strong></p><p>Veuillez désigner un nouvel admin avant de pouvoir supprimer ce membre.</p>`,
+  'cc-orga-member-card.error.last-admin-is-current-user': () => sanitize`<p><strong>Vous êtes le dernier admin de l'organisation</strong></p><p>Veuillez designer un nouvel admin avant de pouvoir modifier votre rôle ou quitter l'organisation.</p>`,
+  'cc-orga-member-card.mfa-disabled': `2FA désactivée`,
+  'cc-orga-member-card.mfa-enabled': `2FA activée`,
+  'cc-orga-member-card.role.accounting': `Comptable`,
+  'cc-orga-member-card.role.admin': `Admin`,
+  'cc-orga-member-card.role.developer': `Développeur`,
+  'cc-orga-member-card.role.label': `Rôle`,
+  'cc-orga-member-card.role.manager': `Manager`,
+  //#endregion
+  //#region cc-orga-member-list
+  'cc-orga-member-list.delete.error': ({ memberIdentity }) => sanitize`Une erreur est survenue lors la suppression de <strong>${memberIdentity}</strong>.`,
+  'cc-orga-member-list.delete.error-unauthorised': ({ memberIdentity }) => sanitize`Vous n'êtes pas autorisé à supprimer <strong>${memberIdentity}</strong> de cette organisation.`,
+  'cc-orga-member-list.delete.success': ({ memberIdentity }) => sanitize`<strong>${memberIdentity}</strong> a été supprimé de l'organisation.`,
+  'cc-orga-member-list.edit.error': ({ memberIdentity }) => sanitize`Une erreur est survenue lors de la modification de <strong>${memberIdentity}</strong>.`,
+  'cc-orga-member-list.edit.error-unauthorised': ({ memberIdentity }) => sanitize`Vous n'êtes pas autorisé à modifier le membre <strong>${memberIdentity}</strong>.`,
+  'cc-orga-member-list.edit.success': ({ memberIdentity }) => sanitize`Le rôle de <strong>${memberIdentity}</strong> a bien été modifié.`,
+  'cc-orga-member-list.error': `Une erreur est survenue pendant le chargement de la liste des membres.`,
+  'cc-orga-member-list.filter.mfa': `Seulement 2FA désactivée`,
```

Comment:
I don't know if I'm the only one but it sounds very weird in french :thinking:

Reply:
I agree, if we invert the words, does it sound better ? "2FA désactivée seulement" ?
I don't have other ideas right now :roll_eyes:

Reply:
I like it, a little bit longer but I find it very clear, I'll test it :+1: